src/main/java/cl/transbank/webpay/Options.java

```diff
@@ -3,8 +3,41 @@
 import lombok.*;
 
 @ToString @NoArgsConstructor @AllArgsConstructor
-public class Options {
+public class Options implements Cloneable {
     @Setter @Getter private String commerceCode;
     @Setter @Getter private String apiKey;
     @Setter @Getter private IntegrationType integrationType;
+
+    public Options buildOptions(Options options) {
+        Options alt = clone();
+
+        // if method recive an options then rewrite each property, this is mandatory
```

Comment:
If the method receives an options object then rewrite each property, this is mandatory

Reply:
Fixed